### DISCUSSION TITLE
VST2 and VST3 Presets Not Working

### DISCUSSION THREAD

**Discussion by gatd4** [apohl79/audiogridder#1121] [Help]
the vst2 and vst3 presets are not working for me in AG latest 1.20 beta 15 using parallels mac
the vst2 presets show up when right clicking on the vst but when I select any they do nothing ,nothing changes
the vst3 presets do not show up at all. presets is greyed out on right clicking.
went back to 1.1.1  vst2 presets work but not vst3 still blank
then went to 1.20.b6  to 1.20.b9 and vst2 presets work but not  vst3
then went to 1.20.b10 -b15 and vst2 presets and vst3 presets dont work at all.
so something changed after b9 to stop vst2 presets working but have no idea why vst3 presets dont show or work at all.
by the the way its not just me other friends with similar setups have the same issues 

any help please?
thanks

**Comment by AlphaComposite** [comment]
VST3 presets are still not working. This is a workflow killer.
